.golangci.yml

```diff
@@ -0,0 +1,36 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+
+linters:
+  disable-all: true
+  enable:
+    - unused
+    - gosimple
+    - structcheck
+    - varcheck
+    - ineffassign
+    - deadcode
+    - unconvert
+    # TODO: enable these later
+    # - govet
+    # - gofmt
+    # - errcheck
+    # - staticcheck
+    # - unparam
```

Comment:
Would you please remove these two for now? I don't want to signal a decision that we want them to be enabled.

Reply:
Sure